Remotion/Web/ClientScript/Scripts/ListMenu.ts

```diff
@@ -149,65 +159,59 @@ class ListMenu
 
       if (itemInfo.DiagnosticMetadata)
       {
-        // Do not render empty diagnostic metadata attributes
-        $.each(itemInfo.DiagnosticMetadata, function (key, value)
+        for (const [key, value] of Object.entries (itemInfo.DiagnosticMetadata))
         {
-          if (value === "" || value === null)
+          // Do not render empty diagnostic metadata attributes
+          if (value !== "" && value !== null)
           {
-            delete itemInfo.DiagnosticMetadata[key];
+            item.setAttribute (key, '' + value);
```

Comment:
new code should not rely on concat-to-convert workarounds unless this is really the official an correct way to do this?
@a-ctor

Reply:
There is no such thing as an "official" conversion. There are three options:

 - `'' + value`
 - `value.toString()`
 - `String(value)`

All of them produce the same result, with the exception that `.toString` throws on `null`/`undefined`. So depending on if you want to throw an exception I would either use `'' + ...` or `.toString`

Reply:
toString(), please. it feels the least hacky and TypeScript should make sure we know if it's nullable